conjur/list/list_logic.py

```diff
@@ -0,0 +1,23 @@
+# -*- coding: utf-8 -*-
+
+"""
+ListLogic module
+
+This module is the business logic for executing the list command
+"""
+# pylint: disable=too-few-public-methods
+class ListLogic:
```

Comment:
ListResources?

Reply:
wary to break this convention because other commands (InitLogic, LoginLogic) follow this convention. Not against changing this but want to make sure we understand what this means :) Are we ok with breaking this repo convention?

Reply:
I wouldn't break a convention, but to me who sees is for the first time, list seems awkward.

Reply:
What is awkward about it? What about `ListResourcesLogic`